### DISCUSSION TITLE
Prisma application on cpanel hosted nodejs application

### DISCUSSION THREAD

**Discussion by royhanreza** [prisma/prisma#11667] [Q&A]
I build a express app with prisma, and upload to cpanel hosting, then run this command: 
`npx prisma migrate dev`
`npx prisma migrate deploy`
`npx prisma generate`

and get this same error
![Screen Shot 2022-02-07 at 11 16 43](https://user-images.githubusercontent.com/24818804/152727244-9d576e58-1777-4c3d-a24d-74df977ad6d2.png)

my `schema.prisma` file:
```prisma
generator client {
  provider      = "prisma-client-js"
  binaryTargets = ["native", "debian-openssl-1.1.x"]
}

datasource db {
  provider = "mysql"
  url      = env("DATABASE_URL")
}
```
prisma version:
```
"prisma": "^3.9.1"
"@prisma/client": "^3.9.1"
```
how to fix this?

**Comment by austincrim** [comment]
Hey @royhanreza 👋,

From the given error, it looks like your Linux server doesn't have the required version of `lib6`. Read more about how to upgrade [here](https://askubuntu.com/questions/421642/libc-so-6-version-glibc-2-14-not-found).

**Comment by nkvch** [comment]
did you came up with solution for this?

**Comment by walidboulanouar** [comment]
i have also trouble with this problem when i want to run nestjs application at cpanel

**Reply by janpio** [reply]
Please stop creating new issues about existing discussions. Your comment here is enough to trigger another look.

Austin's reply above still stands:
https://github.com/prisma/prisma/discussions/11667#discussioncomment-2207128

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
